.github/workflows/ci-test.yml

```diff
@@ -12,13 +12,43 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
 
 jobs:
+  # Build with gcc and test p4c on Ubuntu 22.04.
+  test-ubuntu22:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-22.04
+    env:
+      CTEST_PARALLEL_LEVEL: 4
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+
+    - name: ccache
+      uses: hendrikmuhs/ccache-action@v1
+      with:
+        key: test-${{ runner.os }}-gcc
+        max-size: 1000M
+
+    - name: Build (Ubuntu 22.04, GCC)
+      run: |
+        docker build -t p4c --build-arg UBUNTU_VERSION=22.04 --build-arg IMAGE_TYPE=test --build-arg ENABLE_UNIFIED_COMPILATION=ON .
```

Comment:
Do you even need the docker image at this point?

Reply:
Well, good point, probably not.